### PR TITLE
Setup github actions

### PR DESCRIPTION
**Pull request by ultrainstinct30**

### CODE REVIEW

**Review by utsavjnn — APPROVED** [review]

File: .github/workflows/actions.yaml [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+          node-version: ${{ matrix.node-version }}
+          cache: 'npm'
+      - name: Cache build artifacts
+        uses: actions/cache@v3
+        with:
+          path: |
+            **/target
+            ~/.cargo
+          key: aes-gcm-siv-0.0.1  
+      - run: cargo build --release --manifest-path=circom/Cargo.toml && cargo install --path circom/circom
```
Comment (jinankjain) [inline .github/workflows/actions.yaml:47]: Can't we directly do cargo install circom instead of cloning and using that?

Comment (ultrainstinct30) [reply .github/workflows/actions.yaml:47]: I am not sure, I did it the same way as it was in circom-ed25519 repo.

Comment (jinankjain) [reply .github/workflows/actions.yaml:47]: I think we can give it a try just do cargo install circom.

Comment (ultrainstinct30) [reply .github/workflows/actions.yaml:47]: `error: could not find 'circom' in registry 'crates-io' with version '*'`
`circom` is not listed on crate.io

**Review by jinankjain — APPROVED** [review]

**Pull request merged**